04/js/app.js

```diff
@@ -0,0 +1,26 @@
+
+function createRandomNumberArray(min, max) {
+  let arr = [];
+  
+  for(let i = 0; i < 20; i++) {
+    const num = Math.round(Math.random() * (max - min) + min);
+    arr.push(num);
+  }
+
+  return arr;
+};
+
+function getLargestNum(arr) {
+  return arr.sort((a, b) => b - a).slice(0, 10);
```

Comment:
Warto pamięta, że `sort` modyfikuje źródłową tablicę więc nawet `return` nie jest niezbędne ;) 
Chyba, że postanowimy zrobić kopię przed sortowaniem, ale ten temat wykracza poza podstawy JS ;P

PS. Tutaj poruszamy ważny temat. Nie ma co na początku wchodzić w szczegóły bo może to być za dużo i za bardzo namieszamy mentee w głowie, chyba że wiesz, że już on podstawy przeszedł dano temu ;)